CadRevealFbxProvider/UserFriendlyLogException.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace CadRevealFbxProvider.UserFriendlyLogger;
+
+using System;
+
+public class UserFriendlyLogException(string userFriendlyMessage, Exception? innerException = null)
+    : Exception(userFriendlyMessage, innerException);
```

Comment:
This file is a duplicate of `UserFriendlyLogger/UserFriendlyLogException.cs` and should be removed